src/idl_gen_cpp.cpp

```diff
@@ -719,6 +723,71 @@ class CppGenerator : public BaseGenerator {
       }
     }
     code += "  return builder_.Finish();\n}\n\n";
+
+    //Generate a CreateX function with vector types as parameters
+    if (gen_vector_pars) {
+      code += "inline flatbuffers::Offset<" + struct_def.name + "> Create";
+      code += struct_def.name;
```

Comment:
Part of this function is copy from the regular codegen, so I wrapped this part into new function GenSimpleParam and use this new function in regular codegen and new codegen. Is it suitable?